### ISSUE TITLE
Android app crashes at starting after using the  react-native-fcm

### ISSUE THREAD

**Issue by manojbhardwaj** [evollu/react-native-fcm#861]
1. What version of RN and react-native-fcm are you running?
I using latest react-native-fcm
2. What device are you using? (e.g iOS9 emulator, Android 6 device)?
Android 6 device , Android 7 device (Both device)

**Comment by TarunDas005** [comment]
Add config in android/build.gradle

allprojects {
    repositories {
        ...
        configurations.all {
            resolutionStrategy {
                // Add force (11.4.0 is version you want to use)
                force 'com.google.firebase:firebase-messaging:11.4.0'
                force 'com.google.firebase:firebase-core:11.4.0'
                force 'com.google.android.gms:play-services-gcm:11.4.0'
            }
        }
    }
}

**Comment by manojbhardwaj** [comment]
@TarunDas005 Yes i used for resolve this problem https://github.com/evollu/react-native-fcm/issues/857 and after that when my app is launching it's crashes at starting

**Comment by sy92th** [comment]
@manojbhardwaj You need send more information (Log exception)

**Comment by manojbhardwaj** [comment]
ok let me check sir

**Comment by iuraya** [comment]
I'm facing the same problem. My output from  adb logcat is:

com.google.android.apps.gsa.shared.exception.GsaIOException: Error code: 393238 | Buffer overflow, no available space.

**Comment by sy92th** [comment]
You can check your problem in here
https://stackoverflow.com/questions/43164577/looping-error-on-android-emulator/45462655#45462655

**Comment by stanshivam** [comment]
Same issue

**Comment by Madhavan-reactiveworks** [comment]
@TarunDas005 You saved my life. Thanks :)
